src/main/java/com/zenith/Proxy.java

```diff
@@ -156,6 +156,35 @@ public void start() {
             // ensure we are continuously updating the tablist even on servers that don't frequently send updates
             SCHEDULED_EXECUTOR_SERVICE.scheduleAtFixedRate(this::tablistUpdate, 20L, 3L, TimeUnit.SECONDS);
             SCHEDULED_EXECUTOR_SERVICE.submit(this::updatePrioBanStatus);
+            // time warning
+            SCHEDULED_EXECUTOR_SERVICE.scheduleAtFixedRate(() -> {
+                try {
+                    if (isOnlineOn2b2tForAtLeastDuration(Duration.ofSeconds(60))) {
+                        long onlineSeconds = Instant.now().getEpochSecond() - connectTime.getEpochSecond();
+                        if (onlineSeconds > (21600 - (600))) { // 28800 - 600 (6h - 10 minutes)
+                            for (int i=0; i<10; i++) { //Sends the message 10 times, so you don't miss it. It also sends it every minute from then
```

Comment:
I think the amount of messages being sent is a bit excessive, I think just sending one every couple minutes would be more than enough.

Alternatively, we could set [the title, subtitle, or action bar texts](https://github.com/rfresh2/MCProtocolLib/tree/1.20/src/main/java/com/github/steveice10/mc/protocol/packet/ingame/clientbound/title). I'd personally prefer the action bar so its less obtrusive